README.md

```diff
@@ -87,13 +93,26 @@ export INSTALL_MOD_PATH=`realpath ../rootfs/`
 ./nvbuild.sh -i
 ```
 
-9. flash the device(take recomputer-orin-j401 for example)
-```
+9. flash the device (take `recomputer-orin-j401` for example)
+```bash
```

Comment:
It seems that we don't need to add `bash` in here? Any reason?

Reply:
since these are commands executed in the bash console, I see that in this case the syntax highlighting is not much different from the default code formatting. The decision is also yours, I can run another PR with the `bash` formatting removed

Reply:
Please remove it. Thank you~